packages/twilio-run/src/utils/deployInfoCache.ts

```diff
@@ -71,9 +72,15 @@ export function updateDeployInfoCache(
     deployInfoCacheFileName
   );
 
+  if (username in currentDeployInfoCache) {
```

Comment:
Instead of `in` can use use `currentDeployInfoCache[username]` or even `currentDeployInfoCache.hasOwnProperty(username)`? 

The other thing to consider is that right now this would delete even if you don't deploy to that region. Let's say you deploy to `ie1` it would delete the old entry entirely. So I think we should only do that if `region === 'us1'`.